.github/workflows/reusable_pre_release.yaml

```diff
@@ -78,6 +78,11 @@ on:
         required: false
         type: string
         default: "infrastructure_agent/"
+      windows_version:
+        description: "The version of windows server on which build-win-packages runs"
```

Comment:
*nit*: I'd add something like `[Example]: 2022` in the description. Might be confused with passing `windows-2022` or `2022`. Or better, accept `windows-2022` which will make sense when checking args for the caller. `2022` lacks context.

Reply:
Done. Updated `2022` -> `windows-2022`